opensalve/settings.py

```diff
@@ -52,8 +53,15 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    'corsheaders.middleware.CorsMiddleware',
+    'django.middleware.common.CommonMiddleware',
 ]
 
+CORS_ORIGIN_ALLOW_ALL = False
+CORS_ORIGIN_WHITELIST = (
+    "http://localhost:3000/"
```

Comment:
So this is where the client React app is running ?

Reply:
Its the default for react apps. For now let it be port 3000. We can configure it once we are set with both clients. :)